web-api-db-ctrl/postgres/tenant/20230607110100_coi.sql

```diff
@@ -0,0 +1,16 @@
+--  Copyright 2023 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+ALTER TABLE center_of_interest DROP COLUMN is_positive;
+ALTER TABLE interaction DROP COLUMN user_reaction;
```

Comment:
we only ever stored positive cois/interactions, hence we don't need any data migration.